docs/reference/using/cli/gel_instance/gel_instance_backup.rst

```diff
@@ -0,0 +1,65 @@
+.. _ref_cli_gel_instance_backup:
+
+
+===================
+gel instance backup
+===================
+
+Backup a |Gel| instance to disk.
```

Comment:
As far as I know, these commands only work for local project managed (and Docker compose managed) and Gel Cloud instances, right? We should probably say something about that either here or in the description.

Reply:
These work for locally managed (incl WSL) and cloud instances. Docker and remote aren't supported.